packages/vow/README.md

```diff
@@ -56,9 +56,8 @@ resolver.resolve('now you know the answer');
 
 ## Durability
 
-The vow package supports Zones, which are used to integrate Agoric's vat
-upgrade mechanism and `watchPromise`.  To create vow tools that deal with
-durable objects:
+The `@agoric/vat-data/vow.js` module allows Zones to integrate Agoric's vat
+upgrade mechanism.  To create vow tools that deal with durable objects:
```

Comment:
Is `vat-data/vow.js` this still useful, or should one now only import from @agoric/base-zone and @agoric/vow ?

Reply:
I've added `@agoric/vow/vat.js` to take a direct dependency on `@agoric/internal` (to identify SwingSet vat-upgrade promise rejections). Unfortunately, if `@agoric/vow` becomes `@endo/vow`, that will create a circular dependency with the Agoric SDK.

Any suggestions?

Reply:
Sigh. Not at the present time. I'd be happy to see this merged with this issue + a TODO and/or filed issue explaining it, and then to cross that bridge when we migrate it.